docker-compose.yml

```diff
@@ -1,14 +1,16 @@
 version: '3.8'
```

Comment:
please make sure that volumes for db are correctly set up. Try to add something to the database via API and make sure that data is persisted after re-deploing/restarting the container

Reply:
I checked it and data is persisted